### ISSUE TITLE
Powerline fonts not working anymore

### ISSUE THREAD

**Issue by alcappello** [zeit/hyper#3343]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: macOS 10.14.2
- **Hyper.app version**: 2.1.0
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: https://gist.github.com/alcappello/3f24a69764b6d608eb92f0f559a3e57f
- **Relevant information from devtools** _(CMD+ALT+I on macOS, CTRL+SHIFT+I elsewhere)_: 
- **The issue is reproducible in vanilla Hyper.app**: Yes

## Issue
I use Fish Shell with Hyper and used to have Hack as Powerline font to display special glyphs. It worked just fine in v2.0, but it seems to be broken on v2.1.0. I tried other Powerline fonts (Fira Code, Roboto Mono, Go Mono) without results. It seems to work on Terminal, here's a couple of screenshots to see the difference:

<img width="572" alt="screenshot 2018-12-17 at 21 15 44" src="https://user-images.githubusercontent.com/15313011/50113750-2480dc80-0243-11e9-8c4e-24fdfec551a7.png">

<img width="568" alt="screenshot 2018-12-17 at 21 15 27" src="https://user-images.githubusercontent.com/15313011/50113761-2a76bd80-0243-11e9-9805-f8dd46584f40.png">

**Comment by saminnet** [comment]
I have the same issue!

**Comment by RXminuS** [comment]
Any idea of where this issue might have popped up? I tried going back through the changelog but nothing popped out to me.

**Comment by batjko** [comment]
So, I installed hyper 2 just now from scratch, then changed the font in preferences (to 'Source Code pro for Powerline') and it worked fine.

Then I installed a theme (using the CLI: `hyper install verminal`, which seems to have succeeded) and now the font is back to what it was before, i.e. without Powerline, no matter what I say in the preferences.

**Comment by batjko** [comment]
Actually, strike the above.
I just read in the theme's README, that I need to add overrides to the preferences, which then allows me to set the font for the theme, e.g.:

```js
module.exports = {
  config: {
    // Overrides for the verminal theme
    verminal: {
      fontFamily: '"Source Code Pro for Powerline"'
    },
    // default font size in pixels for all tabs
    fontSize: 14,
    fontFamily: '"Source Code Pro for Powerline", Menlo, "DejaVu Sans Mono", Consolas, "Lucida Console", monospace',

   // [etc...]
```

Works fine for me now, e.g.
![image](https://user-images.githubusercontent.com/7287076/50654856-20b0c200-0f86-11e9-8561-bf7e6087b7b1.png)

**Comment by saminnet** [comment]
It didn't work for me. I tried reinstalling and tired different themes as well, still same result!

**Comment by alcappello** [comment]
@batjko 
I also tried and it didn't work, I have the same problem even on a fresh install and without themes. The new 3.0.0-canary.1 doesn't resolve the problem. You seem to use Hyper on Windows (I can see the burger menu on the top left corner in your screenshot), I'm on macOS. Possibly related to OS?

**Comment by batjko** [comment]
> You seem to use Hyper on Windows
Indeed. In the screenshot I'm using the WSL, which is what I do most development from.

> Possibly related to OS?

Not idea but yea, possibly. 
I also just learned that not all themes require this separate configuration I mentioned. The Material theme works just fine from the standard properties for font-family.
So looks like my issue was just a Verminal thing.

**Comment by saminnet** [comment]
> @batjko
> I'm on macOS. Possibly related to OS?
I'm on macOS as well and I'm using Fish shell of that's related as well.

**Comment by whatsrtos** [comment]
```
    verminal: {
      fontFamily: '"Fantasque Sans Mono", "Powerline Consolas"',
      fontSize: 14,
    },
```

It works for me, make sure using theme “verminal”

**Comment by Tiesco789** [comment]
In the font family I just added `'"Source Code Pro for Powerline", monospace, powerline'`. This works for me!

**Comment by aalliikk** [comment]
It doesn't work for me. 

I have Hyper 3.0.2 installed on macOS High Sierra. 

Also, I've installed "Source Code Pro for Powerline", but it doesn't change anything. 

What to do?

**Comment by joelsantosjunior** [comment]
It doesn't work for me as well
I have Hyper 3.0.2 on Windows 10 pro
Also, I've installed "Source Code Pro for Powerline", but it doesn't change anything.

**Comment by HoshiYamazaki** [comment]
Its working for me under macOS Mojave 10.14.6 with Powerline fonts installed and also on Windows 10 Home too :)

With mac I was forced to do @batjko solution, on Windows no fix was needed.

**Comment by scott-joe** [comment]
If you're running into this error after upgrading to macOS Catalina, update the permissions on your `sudo chmod 775 ~/Library/Fonts/**/*`. That's what finally fixed this for me.

**Comment by ghazette** [comment]
> Actually, strike the above.
> I just read in the theme's README, that I need to add overrides to the preferences, which then allows me to set the font for the theme, e.g.:
> 
> ```js
> module.exports = {
>   config: {
>     // Overrides for the verminal theme
>     verminal: {
>       fontFamily: '"Source Code Pro for Powerline"'
>     },
>     // default font size in pixels for all tabs
>     fontSize: 14,
>     fontFamily: '"Source Code Pro for Powerline", Menlo, "DejaVu Sans Mono", Consolas, "Lucida Console", monospace',
> 
>    // [etc...]
> ```
> 
> Works fine for me now, e.g.
> ![image](https://user-images.githubusercontent.com/7287076/50654856-20b0c200-0f86-11e9-8561-bf7e6087b7b1.png)

IT WORKS AWESOME
